src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -28,95 +28,108 @@
-                                        {:route          chosen-route
-                                         :token-decimals token-decimals
-                                         :native-token?  native-token?
-                                         :receiver?      true})
-         sender-possible-chain-ids     (map :chain-id sender-network-values)
-         sender-network-values         (if routes-available?
-                                         (send-utils/network-amounts
-                                          {:network-values
-                                           (if (= tx-type :tx/bridge)
-                                             from-network-amounts-by-chain
-                                             (send-utils/add-zero-values-to-network-values
-                                              from-network-amounts-by-chain
-                                              sender-possible-chain-ids))
-                                           :disabled-chain-ids disabled-from-chain-ids
-                                           :receiver-networks receiver-networks
-                                           :token-networks-ids token-networks-ids
-                                           :from-locked-amounts from-locked-amounts
-                                           :tx-type tx-type
-                                           :receiver? false})
-                                         (send-utils/reset-loading-network-amounts-to-zero
-                                          sender-network-values))
-         receiver-network-values       (if routes-available?
-                                         (send-utils/network-amounts
-                                          {:network-values     to-network-amounts-by-chain
-                                           :disabled-chain-ids disabled-from-chain-ids
-                                           :receiver-networks  receiver-networks
-                                           :token-networks-ids token-networks-ids
-                                           :tx-type            tx-type
-                                           :receiver?          true})
-                                         (cond->
+(defn- add-not-enough-assets-data
+  [send-data chosen-route]
+  (-> send-data
+      (assoc :route                     chosen-route
+             :loading-suggested-routes? false
+             :suggested-routes          {:best []}
+             :enough-assets?            false)
+      (update :sender-network-values send-utils/reset-loading-network-amounts-to-zero)
+      (update :receiver-network-values send-utils/reset-loading-network-amounts-to-zero)))
+
+(rf/reg-event-fx
+ :wallet/suggested-routes-success
+ (fn [{:keys [db]} [suggested-routes-data enough-assets?]]
+   (if-not enough-assets?
+     {:db (update-in db [:wallet :ui :send] add-not-enough-assets-data (:best suggested-routes-data))}
```

Comment:
This is the call to add a specific db state when there're enough assets to pay gas fees.

We needed to implement a different state because previously this case was considered an error and the route wasn't returned, we need the route in order to extract the highlighted data here:

![image](https://github.com/user-attachments/assets/1bac9ce1-5ce8-4be1-a477-e0af10fdf939)